### PR TITLE
#181 make UnirestException a RuntimeException

### PR DESCRIPTION
**Pull request by mmaryo**

### CODE REVIEW

File: src/main/java/com/mashape/unirest/request/BaseRequest.java [hunk 1]
```diff
@@ -52,7 +51,7 @@ protected BaseRequest() {
 		super();
 	}
 
-	public HttpResponse<String> asString() throws UnirestException {
+	public HttpResponse<String> asString() {
```
Comment (ryber) [inline src/main/java/com/mashape/unirest/request/BaseRequest.java]: If you wanted to make this as small a change as possible you could leave the throws. It doesn't hurt anything to declare what RuntimeExceptions you throw and I personally think it's odd that we don't do that more in Java. It's not functional but it is a form of documentation. Or at least it makes the PR smaller. I'll leave it up to the library maintainers to make the call through.

**Comment by mmaryo** [comment]
@shashiranjan84 
not push since 9 month
any plan for the next ?

**Comment by ryber** [comment]
That's funny, I just tweeted to Mashape an hour ago about the same thing. https://twitter.com/ryber/status/812715901590130688

**Comment by ryber** [comment]
Hello, this should be resolved or addressed in Unirest 2. Please see the [upgrade guide](https://github.com/Kong/unirest-java/blob/master/UPGRADE_GUIDE.md) for important changes.

I merged in another Fork to supply this and the code is now drastically different so this can't merge anyway. In any case thank you so much for your time and patience. I hope as one of the new maintainers we can more responsive in the future.

**Pull request closed without merging**